zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -309,6 +309,28 @@ public void testAutoRestartInterpreterAfterSchedule() throws InterruptedExceptio
   }
 
   @Test
+  public void testExportAndImportNote() throws IOException, CloneNotSupportedException,
+          InterruptedException {
+    Note note = notebook.createNote();
+    note.getNoteReplLoader().setInterpreters(factory.getDefaultInterpreterSettingList());
+
+    final Paragraph p = note.addParagraph();
+    String simpleText = "hello world";
+    p.setText(simpleText);
```

Comment:
@swkimme 
I was trying to say, the test doesn't test import note when p.getReturn() is `String`. 
However this unittest is still good, too. Anyway we didn't have test for import/export function.

About changing Job.result into InterpreterResult, we'll need to consider import note.json created from previous version of Zeppelin. They'll still have Job.result with string type.